src/components/ForgotPassword.jsx

```diff
@@ -0,0 +1,50 @@
+// import React from 'react'
+// import FaArrowRightLong from 'react-icons';
+import { Link } from 'react-router-dom';
+
+const SignUp = () => {
+    return (
+        <main className="w-full h-screen bg-black">
+            <section className="md:flex h-screen">
+                <div className=" md:w-6/12 hidden md:flex flex-col text-center bg-[#10151B] pt-20 text-white ">
+                    <h1 className="text-3xl font-bold ">Forgot Password</h1>
+                    <div className=" flex justify-center items-center">
+                        <img
+                            src="/src/assets/forgetPassword.png"
+                            alt="sign up illustration"
+                            className="max-w-lg"
+                        />
+                    </div>
+                </div>
+                <form className="md:w-6/12 text-white bg-black pt-40   px-5 md:px-0  ">
+                    <h1 className="text-3xl font-bold md:hidden mb-8 text-center">
```

Comment:
In this case, using multiple h1s in a single page might not be a great idea for SEO and accessibility 

https://www.greenlanemarketing.com/resources/articles/seo-101-seo-and-multiple-h1-tags/#:~:text=The%20Truth%20About%20Multiple%20H1%20Tags%20and%20SEO&text=Our%20conclusion%20is%20that%20while,the%20structure%20of%20the%20page.